src/include/sof/drivers/edma.h

```diff
@@ -73,8 +73,6 @@
 #define EDMA_TCD_ATTR_DSIZE_32BYTE	0x0005
```

Comment:
@tlauda what dmac stands for? dma controller?

Reply:
@dbaluta I would assume so.

Reply:
Ok, I'm fine with this change.